lib/versioning/VersionID.js

```diff
@@ -116,12 +116,19 @@ function generateVersionId(info, replicationGroupId) {
     lastSeq = (lastTimestamp === ts) ? lastSeq + 1 : 0;
     lastTimestamp = ts;
 
+    // if USE_HEX_VERSION_ID is set, info is used. By default, it is not used.
+    if (process.env.S3_VERSION_ID_ENCODING_TYPE === 'hex') {
+        // info field stays as is
+    } else {
+        info = ''; // eslint-disable-line
+    }
```

Comment:
```suggestion
    if (process.env.S3_VERSION_ID_ENCODING_TYPE !== 'hex') {
        info = ''; // eslint-disable-line
    }
```
Having an empty if looks weird to me, I would flip the condition

Reply:
I wrote it this way to be explicit about what's happening and to avoid a negative predicate check. I agree it looks weird, but I feel the behavior is obfuscated the other way.